.dockerignore

```diff
@@ -0,0 +1,9 @@
+.env
+.vscode/
+.github/
+.git/
+
```

Comment:
Minor extra one I quite like here: `Dockerfile`. Arguably it's quite nice being able to find the Dockerfile inside the container so you can see how it was created, but whilst you're actually making changes having it ignored means you invalidate the cache less. That said, this is all quite theoretical because you're mostly taking a whitelist-approach (i.e. you don't just `COPY . .`) so it won't actually make a change to yours!